### PR TITLE
Bug: No Contacts Error Fix

### PR DESCRIPTION
**Pull request by sam-warren**
Card: https://github.com/bcgov/cas-cif/issues/1496

Added check for schema.anyOf in widget, now if there are no entries for the search widget, it will render "No options" like below:

Screenshot from 2023-04-14 13-15-33

### CODE REVIEW

File: app/tests/unit/components/ProjectRevision/routinelyUpdatedFieldFactory.test.tsx [hunk 1]
```diff
@@ -219,3 +219,30 @@ describe("The routinely updated field factory", () => {
     expect(logSpy).toHaveBeenCalledWith("update mutation called");
   });
 });
+
+it("should return 'no options' when no schema.anyOf does not exist or is empty", async () => {
```
Comment (BCerki) [inline app/tests/unit/components/ProjectRevision/routinelyUpdatedFieldFactory.test.tsx:223]: We're not doing routinely updated fields anymore, so I believe we'll be removing this file at some point as tech debt (@Sepehr-Sobhani, can you confirm?). I'd suggest moving this test into `SearchDropdown.test.tsx` (and changing the filename to `SearchDropdownWidget.test.tsx` if you're already in there)

**Review by BCerki — COMMENTED** [review]
Looking good!

File: app/tests/unit/components/ProjectRevision/routinelyUpdatedFieldFactory.test.tsx (hunk 1 above)
Comment (Sepehr-Sobhani) [reply app/tests/unit/components/ProjectRevision/routinelyUpdatedFieldFactory.test.tsx:223]: Yes, we no longer need `routinelyUpdatedFieldFactory` and its related test.

Comment (sam-warren) [reply app/tests/unit/components/ProjectRevision/routinelyUpdatedFieldFactory.test.tsx:223]: Good catch! Thanks for clarifying :)

**Review by tmastrom — APPROVED** [review]
LGTM

**Review by Sepehr-Sobhani — DISMISSED** [review]

**Pull request merged**